docker/Dockerfile

```diff
@@ -1,34 +1,9 @@
 #Dockerfile for rhq-metrics
 
-FROM jboss/base-jdk:7
+FROM vnguyen/wildfly-jdk8
```

Comment:
I could be more specific with WildFly version.  vnguyen/wildfly-jdk8 image was built from 8.2.0.Final.